guides/source/getting_started.md

```diff
@@ -113,6 +122,38 @@ $ sqlite3 --version
 
 The program should report its version.
 
+#### Installing Node
+
+Since Rails comes with Webpacker as JavaScript pre-processor and bundler,
```

Comment:
I think this is too much detail; the user doesn't need to know anything about Webpacker to complete this guide. We're really just installing Node.js and Yarn so that `rails new` doesn't crash! Perhaps we could combine the two sections, with a brief explanation that they're "for JavaScript", e.g.:

> #### Installing Node.js and Yarn
>
> Finally, you'll need Node.js and Yarn installed to manage your application's JavaScript.

Reply:
I agree, I've just removed unnecessary details.